invenio_communities/assets/semantic-ui/js/invenio_communities/members/components/ErrorMessage.js

```diff
@@ -20,7 +19,9 @@ export class ErrorMessage extends Component {
             {errors.map((error) => {
               return (
                 <Message.Item>
-                  <strong>{error.field}: </strong>
+                  {error.field !== "_schema" && (
```

Comment:
not the pretties, but Im not sure how to fix it otherwise: Marshmallow will always add a field name, `_schema` if not present.

Reply:
Can you add a comment there as a reminder?

Reply:
Done :)